docs/source/data-types/udt.md

```diff
@@ -76,3 +76,33 @@ while let Some((my_type_value,)) = iter.try_next().await? {
 # Ok(())
 # }
 ```
+
+## C-style Enums
+
+With this feature, you can map Rust C-style enums to integer columns (`int`, `smallint`, `tinyint`) in ScyllaDB. The driver macros support this via the `repr` attribute.
+
+When you have a C-style enum (an enum where all variants are unit variants, i.e., they have no fields), you can instruct the driver to serialize it as a specific integer type using `#[scylla(repr = "TYPE")]`.
+
+### Supported Types
+You can use any standard Rust signed integer type: `i8`, `i16`, `i32`, `i64`, etc.
+
+### Example
+
+```rust
+# extern crate scylla;
+# async fn check_only_compiles() {
+use scylla::{SerializeValue, DeserializeValue};
+
+// Defines an enum that will be stored as an 'int' in ScyllaDB.
+#[derive(SerializeValue, DeserializeValue, PartialEq, Debug, Clone, Copy)]
+#[scylla(repr = "i32")]
+#[repr(i32)]
+enum Status {
```

Comment:
:thought_balloon: With current implementation, one needs to duplicate the `repr` attribute. Perhaps it makes more sense to parse `#[repr(<typ>)]` attributes in the macros code instead? Is this even possible?

Reply:
Strongly agreed, if possible we should not introduce new attribute.